lib/entities/integrations/models.ts

```diff
@@ -8,6 +8,8 @@ export type IWebhooksIntegrationData = {
 
 export type ISlackIntegrationData = {
   accessToken: string | null;
+  teamId: string;
+  botUserId: string;
```

Comment:
Does `| null` mean that the value can be null? Does that mean when doing a GET Integration without `showSensitiveData` = false, the teamId and botUserId would error?

Reply:
@Deacon-McIntyre That's a good point, I was wondering why the access token would be null! I'll change this. It looks like the validation at the API point will be able to stop null anything that's invalid so this should be ok to implement into the id properties